applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -491,23 +491,31 @@ void BaseSolidElement::CalculateDampingMatrix(
 
 void BaseSolidElement::CalculateOnIntegrationPoints(
     const Variable<bool>& rVariable,
-    std::vector<bool>& rOutput,
+    std::vector<bool>& rValues,
```

Comment:
Why do you change this?
In the baseclass it is `rOutput`

Reply:
Fuck, one sec

Reply:
Done